### ISSUE TITLE
[Keyboard Navigation - LAPS -Domain Controller- Home Screen] NVDA focus order should be logical on 'change, repair and remove installation' screen on scan mode.

### ISSUE THREAD

**Issue by sonimishra** [wixtoolset/issues#5815]
## Bugs

* Which version of WiX are you building with?
3.7.4522.0 (But I see this bug in the Wix4 repo)

* Describe the problem and the steps to reproduce it.
People who depend on screen reader will not be able to navigate on the screen properly  if narrator focus moves to hidden object in narrator scan mode. 

**Repro Steps:**
1-Go to windows icon and search for remote desktop.
2-Go to the name field and provide the IP address(Domain controller) to connect to the Domain controller VM.
3-Provide valid username and password for successful connection.
4-Go to LAPS named folder in the desktop and select the executable file:- LAPS 64 bit. Right click on this file and select Install button.
5-Navigate to the Run button appearing on the window and select it.
6-Navigate to various controls lying on the LAPS-Change, repair or Remove Installation window opened.
7. Verify the NVDA focus order on 'change, repair and remove installation' screen.

**Actual:** 
NVDA focus order is not logical on 'change, repair and remove installation' screen on scan mode. First it navigates to all the interactive buttons on the screen and then navigate to information below each button.

**Expected:** 
NVDA focus order should be logical on 'change, repair and remove installation' screen on scan mode. After each button, screen reader focus should immediately navigate to information below it.  

**Note:**
Focus order is different in Narrator Scan Mode.

**Test Environment:**
VM Details : 10.130.56.94
OS: Win Server 2016
Version 1607 (OS Build 14393.2068)
Screen reader : NVDA

**Comment by v-tgovil** [comment]
HCL

**Comment by barnson** [comment]
We need more data, like a screen shot showing the problem.

**Comment by barnson** [comment]
Without clarifying data, we cannot act on this issue.
